module/modmanager/manager_test.go

```diff
@@ -215,6 +234,52 @@ func TestModManagerFunctions(t *testing.T) {
 	}
 	err = mgr.Add(ctx, modCfg)
 	test.That(t, err, test.ShouldEqual, errModularResourcesDisabled)
+
+	t.Log("test empty dir for CleanModuleDataDirectory")
```

Comment:
This single "test" called `TestModManagerFunctions` is pretty gross.

If people are interested, I'd be happy to break it out into many `t.Run(...)`s.

However, I'd rather to that in a different PR.

Reply:
Up to you but I'd say defer to another PR. This is one of our only tests that has this `t.Log`-separating-sections design; I don't think it's necessarily worse that a bunch of `t.Run`s.

Reply:
Resolving.